src/components/DeadlineField.tsx

```diff
@@ -51,12 +51,10 @@ export default function DeadlineField(): ReactElement {
             dispatch(updateTransactionDeadline(Deadlines.Custom))
           }
           onChange={(e): void => {
-            let deadlineNumber
-            e.target.value === "" || parseFloat(e.target.value) < 0
-              ? // if left blank, deadline defaults to 10
-                (deadlineNumber = Deadlines.Ten)
-              : (deadlineNumber = parseFloat(e.target.value))
-            dispatch(updateTransactionDeadline(deadlineNumber))
```

Comment:
don't think this was meant to be removed

Reply:
reinserted